frontend/apps/web/app/(mgmt)/[account]/connections/[id]/page.tsx

```diff
@@ -49,15 +54,19 @@ export default function ConnectionPage({ params }: PageProps) {
         variant: 'destructive',
       }),
     extraPageHeading: (
-      <div>
+      <div className="flex flex-row items-center gap-4">
+        <CloneConnectionButton
+          connectionType={data?.connection?.connectionConfig?.config.case}
+          id={data?.connection?.id}
+        />
```

Comment:
minor thing here but let's not render this button unless these data types are present.